### PR TITLE
adding new file

### PR DESCRIPTION
**Pull request by MTHNCOIN**
Adding new folder name (MTHN)

**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): 27645
(https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)

##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/MTHN: 
        
 
  
    
    
    Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (transaction={"to":"0xcC0D48a5530Cca0481105cCD61A14C495A51c901","data":"0x574f2ba3","accessList":null}, code=CALL_EXCEPTION, version=providers/5.7.2)
    Truncated error stack:
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@ethersproject/providers/lib/fallback-provider.js:654:52
        at Array.forEach (<anonymous>)

**Comment by MTHNCOIN** [comment]
> Error while running adapter at projects/MTHN:
> 
> ```
> Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (transaction={"to":"0xcC0D48a5530Cca0481105cCD61A14C495A51c901","data":"0x574f2ba3","accessList":null}, code=CALL_EXCEPTION, version=providers/5.7.2)
> Truncated error stack:
>     at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@ethersproject/providers/lib/fallback-provider.js:654:52
>     at Array.forEach (<anonymous>)
> ```

file updated

**Comment by MTHNCOIN** [comment]
Adding new folder name (MTHN)

NOTE

If you would like to add a volume adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
If you would like to add a liquidations adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.
Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
Please enable "Allow edits by maintainers" while putting up the PR.
Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
Please fill the form below only if the PR is for listing a new protocol else it can be ignored/replaced with reason/details about the PR
For updating listing info It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can edit it there and put up a PR
Do not edit/push package-lock.json file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap
Name (to be shown on DefiLlama): MTHN
Twitter Link: https://twitter.com/Mthncoin
List of audit links if any: https://twitter.com/cyberscope_io/status/1697541115769917654
Website Link: https://www.mthnetwork.io/
Logo (High resolution, will be shown with rounded borders): https://www.mthnetwork.io/images/200%20x%20200%20(1).png
Current TVL:
Treasury Addresses (if the protocol has treasury)
Chain: BSC
Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)
Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): 27645
(https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)

Short Description (to be shown on DefiLlama): The team is developing a comprehensive blockchain ecosystem that incorporates a social media platform, education platform, e-commerce platform, NFT marketplace, and more. MTH Network is working on the Evoke enterprise blockchain to solve the problems of the industry 5.0 through blockchain application. Evoke will be the fourth generation blockchain used on the all projects of MTH Network. MTHN is a utility coin/token on Binance Smart Chain that can be used as a means of payment across all MTH Network platforms.
Token address and ticker if any:
Category (full list at https://defillama.com/categories) *Please choose only one:
Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
forkedFrom (Does your project originate from another project):
methodology (what is being counted as tvl, how is tvl being calculated):
Github org/user (Optional, if your code is open source, we can track activity):

**Comment by realdealshaman** [comment]
@MTHNCOIN Hi, the tvl code you are using indicates that your project is a dex, is MTHN a dex?

**Comment by MTHNCOIN** [comment]
Yes MTHN a dex

File: projects/MTHN/index.js
Comment (g1nt0ki) [inline projects/MTHN/index.js:3]: this is erc20 token address, not a dex

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
.

File: projects/MTHN/index.js
Comment (MTHNCOIN) [reply projects/MTHN/index.js:3]: Its an BEP 20 Token listed on Dex (Pancake Swap)
https://pancakeswap.finance/swap?inputCurrency=0xcC0D48a5530Cca0481105cCD61A14C495A51c901

https://coinmarketcap.com/currencies/mth-network/

Comment (g1nt0ki) [reply projects/MTHN/index.js:3]: yes, but we dont list tokens but defi projects

**Pull request closed without merging**
